roles/azimuth_capi_operator/defaults/main.yml

```diff
@@ -80,7 +80,11 @@ azimuth_capi_operator_capi_helm_worker_root_volume_availability_zone: "{{ azimut
 
 # Options for LoadBalancer services
 # https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md#load-balancer
-azimuth_capi_operator_capi_helm_openstack_loadbalancer_method:
+azimuth_capi_operator_capi_helm_openstack_loadbalancer_method: >-
+  {{-
+    'SOURCE_IP_PORT' if azimuth_capi_operator_capi_helm_openstack_loadbalancer_provider == 'ovn'
```

Comment:
IMHO, putting the condition on the next line, like other examples, improves readability

Reply:
Agreed, fixed